### DISCUSSION TITLE
Manual/Console

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#555] [Notes]
# Manual/Console

https://docs.unity3d.com/Manual/Console.html

**Comment by vertxxyz** [comment]
You can use the [`[HideInCallstack]`](https://docs.unity3d.com/ScriptReference/HideInCallstackAttribute.html) attribute to mark methods to be hidden when **Strip Logging Callstack** is enabled via the Console window menu.
